### PR TITLE
chore: sync issue templates from the cli

### PR DESCRIPTION
**Pull request by asalkeld**
part of nitrictech/nitric#248

### CODE REVIEW

**Review by HomelessDinosaur — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+about: Report a bug encountered while using nitric
+labels: bug
+---
+
+**Describe the bug**
+<!--
+A clear and concise description of what the bug is. What happened?
+-->
+
+**Expected behavior**
```
Comment (HomelessDinosaur) [inline .github/ISSUE_TEMPLATE/bug-report.md:12]: Are we appealing to american english with behavior vs behaviour? Or should this be changed to everywhere else language?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+<!--
+A clear and concise description of what you expected to happen.
+-->
+
+**To Reproduce**
+
+Steps to reproduce the behavior:
```
Comment (HomelessDinosaur) [inline .github/ISSUE_TEMPLATE/bug-report.md:20]: As above

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (asalkeld) [reply .github/ISSUE_TEMPLATE/bug-report.md:12]: This is as written by @elisselockhart - I suspect the answer is yes..

Comment (elisselockhart) [reply .github/ISSUE_TEMPLATE/bug-report.md:12]: This is the spelling that was there previously, but I don't think we've made a conscious decision on which spellings to use as a company.

**Review by elisselockhart — APPROVED** [review]

**Pull request merged**